showcase/app/components/shw/divider.gts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import { concat } from '@ember/helper';
+
+export interface ShwDividerSignature {
+  Blocks: {
+    default: [];
+  };
+  Args: {
+    level?: 2;
+  };
+  Element: HTMLHRElement;
+}
+
+export default class ShwDivider extends Component<ShwDividerSignature> {
+  <template>
+    <hr
+      class="shw-divider {{if @level (concat 'shw-divider--level-' @level)}}"
+      ...attributes
```

Comment:
[question] `...attribute` was not present in the original implementation; why the decision to add it? have you seen instances where it's used?

Reply:
[if you remove it, then I _think_ you can also remove the `Blocks` entry in the signature]

Reply:
I just thought it could be good to add to follow what we've done in most other components. Anyway I'll take it out.

Reply:
Removed it & pushed up